src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2426,10 +2434,98 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="func-lower"
+    title={<InlineCode>lower(str)</InlineCode>}
+  >
+    Use the lower() function transliterates all alphabetic characters of a string value to lower-case.
```

Comment:
_to transliterate_

Reply:
"Transliterate" is kind of fancy, maybe use "change"?

Reply:
I believe "lower case" would be correct here; "lower-case" is for the verb or adjective form.